src/Modal/Actions.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import css from 'styled-jsx/css'
+import cx from 'classnames'
+
+import { Button } from '../index'
+
+export const Actions = ({ children }) => (
+    <div className={cx('actions')}>
+        {children}
+
+        <style jsx>{`
+            .actions {
+                padding: 0 24px 20px;
+                display: flex;
+                justify-content: flex-end;
+            }
+        `}</style>
+    </div>
+)
+
+Actions.propTypes = {
+    children: PropTypes.oneOfType([
+        PropTypes.element,
```

Comment:
I've tried to be consistent in `ui-core` about using `propTypes` to be consistent with how we import defaults from other libs and to indicate a difference between components and other things.

The only exceptions for initial capital letters:
 
- Named imports/exports
- React (the default export)
- Components

Reply:
> Pretty sure that PropTypes.element will only accept React components, not HTML elements.

That's true. I tried to follow [the specs](https://github.com/dhis2/design-system/blob/master/molecules/modal.md). There the `Modal` can only have our `Button` components as children.
Is there a reason why we should switch to `propTypes.node`?

> I think that across the code base we are importing the default as propTypes, not PropTypes

You're totally right! I'm used to `PropTypes` as well, have never seen it in camelCase. For consistency reasons, I'll change that to `propTypes`

Reply:
I can follow your reasoning only up to a point:
- In this case the specs are very specific (and a bit restrictive). If we want to follow them to the letter, then I would argue that rendering children in the Actions is not the way to go, because this is what you do when you want to offer flexibility. If we want to stick to the specs, I think we should not have children, but instead something like this:
```javascript
const actionPropType = propTypes.shape({
    onClick: propTypes.func.isRequired,
    label: propTypes.string.isRequired,
})

Actions.propTypes = {
    primaryAction: actionPropType,
    secondaryAction: actionPropType.isRequired,
    destructiveAction: actionPropType,
}
```

- On the other hand, if we want to offer flexibility and allow rendering children, then I don't see any reason why strictly speaking this needs to be a react component, it might as well be a HTML element

So personally, I would either a) change the implementation to follow the specs more closely, or b) change the children propType to `propType.node`.

Reply:
UPDATE: if we want to keep things flexible, and I think we should, then using `PropTypes.element` is fine. Turns out I had a misconception about that, because I thought it would only accept proper React components and not native elements. I was wrong.